cpp/custrings/tests/test_convert.cu

```diff
@@ -134,8 +136,10 @@ TEST_F(TestConvert, ToFloat)
             double fval2 = expected[idx];
             if( std::isnan(fval1) )
                 EXPECT_TRUE( std::isnan(fval2) );
+            else if( std::isinf(fval1) )
+                EXPECT_TRUE( std::isinf(fval2) );
             else
-                EXPECT_EQ(fval1,fval2);
+                EXPECT_NEAR(fval1,fval2,1e-10);

```

Comment:
Have you tried using EXPECT_DOUBLE_EQ here?

Reply:
I tried that initially and it failed. Looks like I can get it to work if I update the expected value.

Reply:
So `EXPECT_DOUBLE_EQ` fails now on 10.1.
I tested `EXPECT_NEAR` on 10.1 and 10.2 so I think it should be ok.